documentation/bootstrap_upgrade_notes.md

```diff
@@ -0,0 +1,47 @@
+# Bootstrap upgrade notes
+
+**NOTE: If you have previously disabled your CA, please read this.**
+
+In the past, puppetserver users who have modified their `bootstrap.cfg` file to disable the CA on their compile masters and support a multi-master setup have been subject to a painful upgrade experience.
```

Comment:
Use Puppet Server instead of puppetserver unless you're referring specifically to the `puppetserver` service.